nncf/common/graph/utils.py

```diff
@@ -68,3 +68,28 @@ def get_first_nodes_of_type(graph: NNCFGraph, op_types: List[str]) -> List[NNCFN
     for root in graph_roots:
         first_nodes_of_type.extend(graph.traverse_graph(root, partial_traverse_function))
     return first_nodes_of_type
+
+
+def get_split_axis(input_shapes: List[List[int]], output_shapes: List[List[int]]) -> int:
+    """
+    Returns split/chunk axis by given input and output shape of split/chunk node.
+
+    :param input_shapes:
+    :param output_shapes:
+    :returns: Split/Chunk axis of given split/chunk node.
+    """
+    axis = None
+    none_dim = None
```

Comment:
`none_dim` looks redundant

Reply:
In my thought, `none_dim` is necessary in the clause `if axis is None:`. Do you think this clause is unnecessary? If there are no exceptions at all, I think it would be better to remove this clause altogether. If `axis` can't figure it out above, it seems necessary to make an exception through this section.

Reply:
My concern is that `none_dim` is constant None, it isn't modified anywhere here

Reply:
I remove for `none_dim` in here. Actually, I added it cause `concat` case consider `none_dim` case, but I think, `none_dim` is not needed in `chunk` case.